.github/workflows/code-health-tools.yml

```diff
@@ -25,6 +25,31 @@ jobs:
       run: |
         pushd tools/cli
         make build
+  unit-tests:
+    needs: build
+    env:
+      COVERAGE: coverage.out
+      TEST_CMD: gotestsum --junitfile unit-tests.xml --format standard-verbose --
+      UNIT_TAGS: unit
+      INTEGRATION_TAGS: integration
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+      - name: Install Go
+        uses: actions/setup-go@0c52d547c9bc32b1aa3301fd7a9cb496313a4491
+        with:
+          go-version-file: 'tools/cli/go.mod'
+      - name: Install gotest
+        run: go install gotest.tools/gotestsum@ec99a250836f069a524bb9d9b5de0a7a96334ea7
```

Comment:
we use this in the cli to have the test summary with failures since gotestsum can produce xunit reports
feel free to use the simpler `go test` command but I wonder if there's a nicer way to have a summary report of tests

Reply:
this https://github.com/smartystreets/goconvey may be a bit too much. xD

Reply:
yeah I don't have a strong opinion I feel in general GH actions lack a good way to represent test summaries, I don't quite like it on the CLI currently and I haven't really found something good enough to what I would expect (a list of tests and the result)